lib/anyway/ext/value_serializer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Anyway
+  module Ext
+    # Extend Object through refinements
```

Comment:
?

Reply:
Btw, we can implement serializer through refinements (as String extension, i.e. `val.serialize`).

Reply:
Copy-paste issue. Yes, I'll update the implementation to use refinements.